### PR TITLE
Create top navbar and partials

### LINKED ISSUE #65: Create top navbar and partials
This will expand upon the user stories 2 and 20 using CSS and partials to display relevant links based on the page the user is currently visiting. The CSS was created in user-story-14 branch.

2. Merchant Dashboard Links
As a merchant,
When I visit my merchant dashboard (/merchants/:merchant_id/dashboard)
Then I see link to my merchant items index (/merchants/:merchant_id/items)
And I see a link to my merchant invoices index (/merchants/:merchant_id/invoices)

20. Admin Dashboard Links
As an admin,
When I visit the admin dashboard (/admin)
Then I see a link to the admin merchants index (/admin/merchants)
And I see a link to the admin invoices index (/admin/invoices)

### PR DESCRIPTION
**Pull request by MWoodshop**
# Describe the changes below:
Uses @westonio menu design from the _admin menu partial
Creates a new partial for visitor menu
Adds admin partial to all admin views
Adds visitor partial to all visitor (non-admin) views
Adds unique menu to root page to select Admin or Visitor to avoid a 404 when accessing the main website*
Adds link to page header text to return to root page
Adds merchant controller and index view page for a user to select their merchant as opposed to having to type it in.*

*Note: I don't see any future user stories addressing this but if there is we can just undo these changes.

# Relevant user story: N/A

# Specific feedback requests? Include file name and line:

# Before submitting, check the following:
- [X] Entire test suite is passing
- [X] SimpleCov test percentage (100%)

# Note any co-authors / anything else you'd like to add:
Discussed in morning standup. Please let me know if anything looks off and I can change/undo anything.

### CODE REVIEW

File: app/views/shared/_admin_topper.html.erb [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
         <li><%= link_to "Invoices", admin_invoices_path %></li>
       <% end %>
     </ul>
   </nav>
-</div>
\ No newline at end of file
+</div>
+<div class="seperator"></div>
+<div class="container">
+</div>
```
Comment (westonio) [inline app/views/shared/_admin_topper.html.erb]: Not sure if it was intended, but these seem unused and possible made it in there as a mistake @MWoodshop

File: app/views/shared/_visitor_topper.html.erb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        <li class="current-menu-item"><%= link_to "My Invoices", merchant_invoices_path %></li>
+      <% else %>
+        <li><%= link_to "My Invoices", merchant_invoices_path %></li>
+      <% end %>
+    </ul>
+  </nav>
+</div>
+<div class="seperator"></div>
+<div class="container">
+</div>
```
Comment (westonio) [inline app/views/shared/_visitor_topper.html.erb]: Same thing here - is there a reason there are `<div>`s without content in them?

**Review by westonio — CHANGES_REQUESTED** [review]

File: app/views/shared/_admin_topper.html.erb (hunk 1 above)
Comment (MWoodshop) [reply app/views/shared/_admin_topper.html.erb]: @westonio Lines 1-5 are adding a clickable link to the main header "Little Esty Shop" to get back to the root page. We can remove before final submission if we don't want it but at least for debugging purposes it lets get to the main page from anywhere.

Lines 25-28 follows the wireframe of having a horizonal bar under the header and nav bar and before the content of eachpage.

Comment (MWoodshop) [reply app/views/shared/_admin_topper.html.erb]: Actually the <div class="container"></div> is no longer being used. I'll remove that. The separator is still being used however.

Comment (westonio) [reply app/views/shared/_admin_topper.html.erb]: Oh gotcha - I just wanted to make sure it wasn't an accident or erroneous.  If it's intended -- no worries, I will approve.

**Review by westonio — APPROVED** [review]

**Review by dani-wilson — APPROVED** [review]
Looks nice!

**Pull request merged**